services/workflows-service/scripts/workflows/dynamic-ui-workflow.ts

```diff
@@ -263,6 +275,11 @@ export const dynamicUiWorkflowDefinition = {
         persistenceStates: ['kyc_manual_review'],
         deliverEvent: 'KYC_RESPONDED',
       },
+      {
+        definitionId: kycEmailSessionDefinition.name,
+        persistenceStates: ['revision_email_sent'],
```

Comment:
It probably should be renamed, since not necessarily something is being persisted - maybe "callbackStates"